Resources/services.xml

```diff
@@ -40,6 +40,12 @@
             <argument type="service" id="mollie_shopware.components.logger"/>
         </service>
 
+        <service class="MollieShopware\Command\OrdersRefundCommand">
+            <tag name="console.command" command="mollie:orders:refund"/>
```

Comment:
do we have the command duplicated in the php and xml? just thinking about what is used :D 
maybe you can just do it in 1 place..(i think in php for consistency?)